src/bun_js.zig

```diff
@@ -48,6 +48,8 @@ pub const Run = struct {
         JSC.markBinding(@src());
         @import("bun.js/javascript_core_c_api.zig").JSCInitialize();
 
+        _ = C.setTtyOrigTermiosIfNeeded();
```

Comment:
I'm not sure we should set this globally on start. Does Node/libuv do that?

Reply:
Actually you're right, I was thinking we could set it at the beginning at the process so we for sure have the right original config, but I think if we always go through the same entrypoint for raw mode, then we can always get it lazily. I'll change that